pkg/userdata/flatcar/provider.go

```diff
@@ -164,7 +168,7 @@ systemd:
             Environment=ALL_PROXY={{ .HTTPProxy }}
 {{- end }}
 
-    - name: download-healthcheck-script.service
+    - name: download-script.service
```

Comment:
Why we renamed this service? AFAIK, it downloads the health-check script, so mentioning it might make sense.

Reply:
Because it's "download all stuff", not just health-check
`ExecStart=/opt/bin/download.sh`